forge-core/src/main/kotlin/io/github/hellocuriosity/TryExt.kt

```diff
@@ -1,9 +1,12 @@
 package io.github.hellocuriosity
 
-@Suppress("SwallowedException")
+@Suppress("SwallowedException", "TooGenericExceptionCaught", "PrintStackTrace")
 fun <T> Any.attempt(block: () -> T) = try {
     block()
 } catch (e: InstantiationError) {
     // Throw model forge exception instead
-    throw ModelForgeException("$this is not yet supported")
+    throw ModelForgeException("$this is not yet supported", e)
```

Comment:
```suggestion
    throw ModelForgeException("${e.message} is not yet supported", e)
```

This would give us the actual intended functionality I was going for way back when, resulting in:

```
io.github.hellocuriosity.ModelForgeException: kotlin.random.Random is not yet supported
```

as opposed to the current cryptic output:

```
io.github.hellocuriosity.ModelForgeException:
io.github.hellocuriosity.ModelForge@1cf56f0f is not yet supported
```